uwblueprint/abtc
# Sign Up Post Request to Backend

## Notion ticket link

[Sign Up Post Request to Backend](https://www.notion.so/uwblueprintexecs/Sign-Up-Post-Request-to-Backend-154810dfa8d347bdbc7bc60a738959ff?pvs=4)



## Implementation description
* Add phone number and emergency phone number to user model
* Link the sign up components as a multistep form
* Update functions to reflect changes in user model
* Make a post request with the sign up info



## Steps to test
1. Navigate to /signup
2. Follow the steps to sign up
3. Verify in MongoDB test.users the new user is created



## What should reviewers focus on?
* Design adheres with Figma design
* State is preserved when going back in sign up process (note: will require confirming email each time going back)
* Appropriate alert shows when user is successfully created in MongoDB or if there is an error


## Checklist
